udfs/remote_udfs/nlp/call_nlp/main.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from google.cloud import language_v1
```

Comment:
Add requirements.txt file to the call_nlp/ directory

File should contain: 
`google-cloud-language==2.6.1`

Reply:
added

Reply:
Please push the requirements.txt file. it's not included in this PR